edge-util/src/Microsoft.Azure.Devices.Edge.Util/edged/version_2019_01_30/generatedCode/HttpWorkloadClient.cs

```diff
@@ -1,45 +1,125 @@
 //----------------------
 // <auto-generated>
-//     Generated using the NSwag toolchain v11.17.6.0 (NJsonSchema v9.10.46.0 (Newtonsoft.Json v9.0.0.0)) (http://NSwag.org)
+//     Generated using the NSwag toolchain v13.3.0.0 (NJsonSchema v10.1.11.0 (Newtonsoft.Json v12.0.0.0)) (http://NSwag.org)
```

Comment:
This code was regenerated with nswag's dotnet 3 option.

Reply:
Same comments as the other generated file, please make sure it uses System.Net.WebUtility.UrlEncode.